js/main.js

```diff
@@ -0,0 +1,67 @@
+let books;
+
+// loader
+function loader() {
+  document.getElementById('view').innerHTML = '';
+  const warning = document.querySelector('.warning');
+  warning.style.visibility = 'hidden';
+  let article = '';
+  books.forEach((book) => {
+    article += `
+            <article>
+             <p class="book-title">${book.title}</p>
+             <p class="author">${book.author}</p>
+             <button class="remove" onclick="whatButton(this)">Remove</button>
+             <hr>
+              </article>
+    `;
+  });
+  document.getElementById('view').innerHTML = article;
+}
+
+if (books === []) {
+  localStorage.setItem('books', JSON.stringify(books));
+} else {
+  books = JSON.parse(localStorage.getItem('books'));
+  loader();
+}
+
+// add a Book fn
+function addToBookCollection(title, author) {
+  books = JSON.parse(localStorage.getItem('books'));
+  const book = {};
+  book.title = title;
+  book.author = author;
+  books.push(book);
+  localStorage.setItem('books', JSON.stringify(books));
+}
```

Comment:
- [ ] after adjusting the previous comment it runs correctly until the user tries to add a book, please have a look at the below error and make sure it is fixed before requesting another code review.
![Screenshot 2022-01-04 114426](https://user-images.githubusercontent.com/39240820/148047614-e27ebe43-b25f-4065-997d-940254aafa82.png)

Reply:
Done 👍